### ISSUE TITLE
MetroDateTime write manually

### ISSUE THREAD

**Issue by yussel53** [thielj/MetroFramework#132]
I want to type MetroDateTime manually, Without Check box, How do I do that

**Comment by ThatLadLegend** [comment]
I think you want to do 
`var dt = DateTime.Now.ToString("h: mm tt");`
`Label.Text = dt;`

To figure out what to put in place of `h: mm tt` go to [this website](https://www.c-sharpcorner.com/blogs/date-and-time-format-in-c-sharp-programming1). It should have all the formats you want.

Hopefully this is what you wanted.

**Comment by yussel53** [comment]
What I really want is to have the cursor in the textbox

**Comment by ThatLadLegend** [comment]
Could you show an example of what you want?

**Comment by yussel53** [comment]
I want to be able to type like in the masked textbox

**Comment by ThatLadLegend** [comment]
So you want to manually type in a date into a textbox?

**Comment by yussel53** [comment]
No, I want to manually enter data into the same MetroDateTime textbox.

**Comment by ThatLadLegend** [comment]
I don’t think you can manually enter the date into a MetroDateTime box. You’re talking about using it like a MaskedText Box I imagine.

**Comment by yussel53** [comment]
Yessss... like MaskedText.. 
Actually accessible but not very comfortable
